src/main/java/org/jabref/gui/groups/GroupDialogViewModel.java

```diff
@@ -386,7 +385,9 @@ public void setValues() {
             descriptionProperty.setValue(editedGroup.getDescription().orElse(""));
             iconProperty.setValue(editedGroup.getIconName().orElse(""));
             groupHierarchySelectedProperty.setValue(editedGroup.getHierarchicalContext());
+            typeExplicitProperty.setValue(true);
 
+            /*
```

Comment:
Is this all obsolete?

Reply:
Hi @Siedlerchr , this is outdated.
I have changed my solution to fix this issue.